ios/Classes/NetIamyellowTiwsWSProxy.m

```diff
@@ -121,8 +124,9 @@ -(void)close:(id)args
 
 -(void)send:(id)msg
 {
-    ENSURE_SINGLE_ARG(msg, NSString);
-
+    // not sure why tianium is passing a __NSArrayM with inside the TiBLob
+    if (![msg isKindOfClass:[NSString class]])
+        msg = [msg[0] data];
```

Comment:
I guess it's written somewhere in the Ti docs. It's odd though because for android it's directly a TiBlob, as expected.